CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# CorisxTH Contributing Guidelines #
+
+## Tell other devs what you are working on! ##
+
+When you start working on a fix/enhancement for an open issue please always
+post a comment in this issue's discussion to tell other devs that you have
+started working on it so that they won't waste any of their free time by
+working on their own fix/enhancement for it.
+
+If there isn't an open issue discussion for a bug fix/enhancement you want
+to work on please check that there isn't a closed discussion for it because
+a previous developer of it didn't finish their work which you could finish.
+
+And if there is no existing issue discussion for what you want to work on
+please open a new issue for it and tell other devs here that your going to/
+have started working on it.
+
+## Enhancement/Suggestions ##
+If you want to make a suggestion, please use the *Feature Request* option when making a new issue. It will provide all the information needed to evaluate the suggestion or enhancement.\
+![image](https://user-images.githubusercontent.com/20030128/123638352-fc609400-d816-11eb-88ad-11030e041fc4.png)
+
+
+## Pull Requests ##
+
+When providing a pull request ensure the description field describes the major
+changes from the request. If your pull request relates to any existing
+issues, include those issues in the description. The pull request provides a default layout to help you fill out this information.
+
+#### Pull Request Statuses ####
+
+CorsixTH uses the following labels inside your Pull Request's title to denote its current progress.
+| Label | Description |
+| ----- | ----------- |
+| **[WIP]** | Work in Progress |
+| **[RFC]** | Request for Comments (aka Ready for Review) |
+| **[RDY]** | Ready |
+
+Please make sure you label and update your Pull Request accordingly. Alternatively, you can submit a **Draft Pull Request** as an alternative to the **[WIP]** label.
+
```

Comment:
Add a sentence that pull requests without label are considered ready for review and merge if found ok.

Reply:
I don't need the [RDY] label anymore, we can drop that as a concept - it just clutters the git history with [RDY] 😄

Reply:
I would disagree -- RDY is very handy as a concept. I know exactly what state a PR is in without going into it.

Reply:
(Not that people listen to us 🙃 anyway)

Reply:
I would rather mark all other states draft, wip, and leave ready blank.

Reply:
What about RFC?

Reply:
Sure, RFC serves a purpose - please look at me, but don't merge.

Reply:
Done. Assuming nobody is gonna be punished for still using [RDY] if they wanted though. That was my only holdup.